### DISCUSSION TITLE
Change account email address?

### DISCUSSION THREAD

**Discussion by TFB60** [aclap-dev/video-downloadhelper#394] [Help and Q&A]
### Your question

My licence for Video Downloadhelper is over 12 months old, and since I originally purchased the software, my email has changed. The old email address is no longer an active domain. How do I change the email address on my account so I can register an existing license?

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: locked
key (only 16 first characters): 05ec7470c32c5293
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Tom",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by jcfrog** [comment]
hi,
please send us an email at licenses@downloadhelper.net
